pom.xml

```diff
@@ -23,6 +26,39 @@
     </properties>
 
     <profiles>
+        <profile>
+            <id>AllScenarios</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <properties>
+                <suite.file>src/test/resources/commonScenarioTestSuite.xml
```

Comment:
It will be better to organize suites files in the resource folder by folder as packages for the homework